lua/entities/hl_camera_ghost/cl_init.lua

```diff
@@ -0,0 +1,22 @@
+include("shared.lua")
+
+local function CheckIfCanDraw() -- copypasted from the advcamera
```

Comment:
Again, rename to `ENT:ShouldDraw`.

I wonder if there's a nice/pretty way to use the same function as in the advanced camera, without resorting to copy-paste (since it's a bad practice).